config/package_layout.json

```diff
@@ -18,13 +18,9 @@
     "config": "config",
     "ld_logger": "ld_logger",
     "codechecker_modules": "cc_lib/python2.7",
-    "codechecker_lib": "cc_lib/python2.7/codechecker_lib",
-    "codechecker_gen": "cc_lib/python2.7/codechecker_gen",
-    "codechecker_db_model": "cc_lib/python2.7/db_model",
-    "codechecker_db_migrate": "cc_lib/python2.7/db_migrate",
-    "storage_server_modules": "cc_lib/python2.7/storage_server",
-    "viewer_server_modules": "cc_lib/python2.7/viewer_server",
-    "cmdline_client": "cc_lib/python2.7/cmdline_client"
+    "codechecker_lib": "lib/python2.7/libcodechecker",
```

Comment:
I think we can rename the `codechecker_lib` to `libcodechecker` and `codechecker_gen` to `gencodechecker` here in the json. Please update the package builder too!